hydra-plutus/src/Hydra/Data/ContestationPeriod.hs

```diff
@@ -63,3 +66,8 @@ posixFromUTCTime utcTime =
   -- since epoch with picosecond precision.
   posixSeconds :: Pico
   posixSeconds = nominalDiffTimeToSeconds $ utcTimeToPOSIXSeconds utcTime
+
+-- NOTE: hardcoded value set to one hour and one minute
+-- so it is higher than what the arbitrary instance can produce
```

Comment:
This is a weird explanation. The contestation period can be 1 week? Also the acceptable difference has nothing to do with the `ContestationPeriod`, but rather is a property of the close transaction. Move it somewhere more approriate.

Reply:
yeah as mentioned, all of this was caused by the wrong check in the validator. Now this is part of the contract validator and we can tweak it to our liking.